moveit_ros/visualization/robot_state_rviz_plugin/include/moveit/robot_state_rviz_plugin/robot_state_display.h

```diff
@@ -112,7 +112,7 @@ private Q_SLOTS:
   void fixedFrameChanged() override;
 
   // render the robot
-  rclcpp::Node::SharedPtr node_;
+  const rclcpp::Node::SharedPtr node_ = rclcpp::Node::make_shared("robot_state_display");
```

Comment:
This should also be defined in the constructor IMO

Reply:
We don't need to define our own node because one is available in the `context_` member. see interactive_marker_display.cpp